src/aiida/cmdline/utils/common.py

```diff
@@ -486,13 +486,14 @@ def build_entries(ports):
         echo.echo(style('\nExit codes that invalidate the cache are marked in bold red.\n', italic=True))
 
 
-def generate_validate_output_file(
-    output_file: Path | None, entity_label: str, appendix: str = '', overwrite: bool = False
+def validate_output_filename(
+    output_file: Path | None,
+    overwrite: bool = False,
 ):
-    """Generate default output filename for `Code`/`Computer` export and validate."""
+    """Validate output filename."""
 
     if output_file is None:
-        output_file = Path(f'{entity_label}{appendix}.yml')
+        raise ValueError('Output filename must be passed for validation.')
```

Comment:
I am a bit missing if these changes in this function are necessary or just to make handling convenient?

Reply:
Not really necessary. I added this function in the past just for convenience to handle the default output file naming and the validation, as that was otherwise duplicated code for `computer` and `code` `export`. Now, I ended up splitting it, and realized that the default name creation doesn't really warrant its own dedicated function. For the `code`, it could even be fully removed, as the `data_export` function implements some validation already. Though, as `Computer` is not a class derived from `Data`, one cannot use `data_export` there, so I left the validation function there, just to keep the changes in this PR simple.